src/adapter.js

```diff
@@ -0,0 +1,27 @@
+// API Blueprint parser for Fury.js
+
+import deckardcain from 'deckardcain';
+import protagonist from 'protagonist';
+
+export const name = 'api-blueprint';
+export const mediaTypes = [
+  'text/vnd.apiblueprint',
+];
+
+export function detect(source) {
+  return mediaTypes.indexOf(deckardcain.identify(source)) !== -1;
+}
+
+/*
+ * Parse an API Blueprint into refract elements.
+ */
+export function parse({source, generateSourceMap}, done) {
```

Comment:
@pksunkara, Fury is meant as a general purpose, format-agnostic API SDK, so the interface to `parse` is using the term that I think is best based on the evidence below (emphasis mine):

> When you build for production, along with minifying and combining your JavaScript files, you **generate** a source map which holds information about your original files.
> ...
> How do I **generate** a source map?
> http://www.html5rocks.com/en/tutorials/developertools/sourcemaps/

.

> The idea is that when you build for production, along with minifying and combining your JavaScript files, you **generate** a source map that holds information about your original files.
> ...
> This line, usually added by whatever tool **generated** the map, is what enables DevTools to associate minified with unminified files.
> https://developer.chrome.com/devtools/docs/javascript-debugging

.

> This is a library to **generate** and consume the source map format described here.
> https://github.com/mozilla/source-map/

.

> In this blog post you’re going to learn how source maps work and take a look at how to **generate** them.
> http://blog.teamtreehouse.com/introduction-source-maps

. 

> Tools that **generate** source maps
> https://github.com/ryanseddon/source-map/wiki/Source-maps%3A-languages,-tools-and-other-info

. 

> CoffeeScript 1.6.1 and above include support for **generating** source maps, a way to tell your JavaScript engine what part of your CoffeeScript program matches up with the code being evaluated.
> http://coffeescript.org/#source-maps

. 

> Specify an output file where to **generate** source map.
> https://github.com/mishoo/UglifyJS2

You'll also notice that in all those examples the words `source` and `map` are always two different words, which is why I'm using `generateSourceMap` instead of `generateSourcemap`. Hopefully that answers the question! :smile: